src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -85,6 +85,9 @@
           </item>
           <item>
            <widget class="QLineEdit" name="memorySamplingPeriodMsLineEdit">
+            <property name="accessibleName">
+             <string>MemorySamplingPeriodEdit</string>
```

Comment:
This "name" looks more like an identifier. If a user was using a screen reader this would be read to them to identify the field. I think we should try to make the name pronouncable for a screen reader and ideally clear to the user. What about just "Memory Sampling Period in Milliseconds" or something similar?

Reply:
To be fair, this is only following what we are doing in other places as well, because our accessible names are mainly used for E2E testing - it's unfortunately not realistic (especially for the capture window) that Orbit can be used reasonably with a screen reader in the near future.

Also, there is a difference between accessible name and accessible description (which we are leaving blank...), though I don't know how they are commonly used by screen readers.

I do get the point, so if we change this to be more in line with the intended use, we should try to consistently do this for the other elements as well in the future.

Reply:
I wasn't aware that we have similar identifiers already in other places. Under these circumstances it's probably best to leave it as-is.